.github/workflows/validate-issues.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   check-labels-on-issues:
-    uses: Automattic/dangermattic/.github/workflows/reusable-check-labels-on-issues.yml@iangmaia/gha-labels-on-issues-check
+    uses: Automattic/dangermattic/.github/workflows/reusable-check-labels-on-issues.yml@trunk
```

Comment:
How do you feel about pinning to a version?

Or... maybe using `trunk` is cool because it will auto-update? 

Then again, auto-updates can backfire if something was to change.

🤔 🤔 🤔

Reply:
Yep, it will auto-update for now. My plan is to keep it in trunk until we ship a first stable version of Dangermattic after all the API changes we've been doing, then we can move to a `@v1` pinning instead.

Reply:
Ah, right. I didn't realize Dangermattic wasn't v1 yet. I should have checked...